compiler/semantic/expr.go

```diff
@@ -436,6 +438,28 @@ func (a *analyzer) semDoubleQuote(d *ast.DoubleQuote) dag.Expr {
 	})
 }
 
+func (a *analyzer) semExists(e *ast.Exists) dag.Expr {
+	q := a.semSeqExpr(e.Body)
+	// Append collect(this) to ensure array of results is returned.
+	q.Body = append(q.Body,
+		&dag.Aggregate{
```

Comment:
We only need to see one value.

```suggestion
	q.Body = append(q.Body,
		&dag.Head{Kind: "Head", Count: 1},
		&dag.Aggregate{
```